demos/graph-classification/supervised-graph-classification.ipynb

```diff
@@ -0,0 +1,559 @@
+{
```

Comment:
<p>The <code>for</code>  loop could be written using <code>index</code>  and indexing:</p><pre class="ql-syntax" spellcheck="false">test_metrics[model.metrics_names.index("acc")]
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1053/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>